.github/CONTRIBUTING.md

```diff
@@ -23,33 +23,33 @@ When submitting a PR, please fill out the template that is presented by GitHub w
 
 ## First Time Setup
 
-1. Install prerequisites:
-   - latest [Visual Studio Code](https://code.visualstudio.com/)
-   - [Node.js](https://nodejs.org/) v8.0.0 or higher
-1. In a terminal:
-
-   ```bash
-   # fork and clone the repository
-   git clone git@github.com:<YOUR-FORK>/Vim.git
-   cd Vim
-
-   # Install the dependencies
-   npm install -g gulp-cli
-   npm install
-
-   # Open in VSCode
-   code .
-
-   # Choose the "Build, Run Extension" in the dropdown of VSCode's
-   # debug tab to build and run the extension.
-   # Or run tests by selecting the appropriate drop down option
-
-   # Alternatively, build and run tests through gulp and npm scripts
-   gulp build                  # build
-   npm test                    # test
-   gulp test                   # run tests inside Docker container
-   gulp test --grep testSuite  # run only tests/suites filtered by js regex inside container
-   ```
+1.  Install prerequisites:
```

Comment:
Do we need all these MD changes?

Reply:
These changes is made by `gulp forceprettier` command.